contents/docs/product-analytics/troubleshooting.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Troubleshooting and FAQs
+---
+
+Having trouble with events? There are a few common reasons that you may not see events appear in your project.
+
+### 1. The library is being used incorrectly. (this is usually the issue)
+
+> Warning: Make sure to not use sensitive data while debugging using third-party tools.
+
+1. Visit [webhook.site](https://webhook.site/) and copy “Your unique URL”.
+2. Inside your code replace [posthog.com](https://app.posthog.com) (or [eu.posthog.com](https://eu.posthog.com) if using the EU Cloud) with “Your unique URL”.
```

Comment:
let's include code snippets for this

Reply:
Maybe something like this:
```suggestion
2. In your PostHog [initialization code](https://posthog.com/docs/product-analytics/installation), replace "https://app.posthog.com" or "https://eu.posthog.com" if using the EU Cloud) with the URL you copied from webhook.site. For example:
```

And then perhaps just include a few code snippets, maybe just web, node, and ruby (I wouldnt include the js-snippet)

<MultiLanguage>

```js-web
posthog.init('<ph_project_api_key>', { api_host: 'https://webhook.site/xxx-yyy-zzz }) // Replace with the URL you copied from webhook.site
```


```node
// node code
```

```ruby
// ruby code
```

</MultiLanguage>

Reply:
added snippets: 
![Screenshot 2023-08-09 at 12 24 24](https://github.com/PostHog/posthog.com/assets/13001502/60926eef-1c8b-4fc2-b901-cac3e03ae092)